### PR TITLE
Figure flowers

### PR DESCRIPTION
**Pull request by yaaHosein**
let us work together!!

### CODE REVIEW

File: src/Flowers.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
-    <img src="C:\Users\Dell\Desktop\my--first-react-app\public\images\SILAS MARNER.jpg" alt="SILAS MARNER"></img>,
-    <h3>GABRIEL OAK</h3>,
-    <p>A magnificent variety, bearing large, many petalled rosette blooms. They are a striking shade of deep pink, the outer petals of each bloom paling slightly over time. The beauty of the blooms is enhanced by a wonderful, strong fruity fragrance. </p>,
-
-    <img src="C:\Users\Dell\Desktop\my--first-react-app\public\images\GABRIEL OAK.jpg" alt="GABRIEL OAK"> </img> ,
-    <h3>HARLOW CARR</h3>,
-    <p>Bears flowers of the most perfect formation – shallow cups of the purest mid pink. They flower very freely from early summer until well into the autumn and have a strong, pure Old Rose fragrance. </p>,
-    <img src="C:\Users\Dell\Desktop\my--first-react-app\public\images\HARLOW CARR.jpg" alt="HARLOW CARR.jpg"></img>]
+    return <>
+        <h3>SILAS MARNER</h3>
```
Comment (SMH110) [inline src/Flowers.js]: You don't need to hardecode any value here 
the value of `h3` should come from the `props` 

you need to pass this information to your component to make re-usable

**Comment by SMH110** [comment]
This is good, you can merge this into main and complete the PR

**Comment by SMH110** [comment]
Have you managed fixing these conflicts?

**Comment by yaaHosein** [comment]
yes I tried 
I wanted to delete every thing in the Flowers.js and App.js files 
which pushed to the main branch
but these were empty 
so I could not

=======
just I cut and past all these tow files codes in figure-flowers branch
but still now I see this error message :

This branch has conflicts that must be resolved
Use the web editor or the  to resolve conflicts.

I do not know why there is conflicting files

**Comment by yaaHosein** [comment]
image


the web editor is not doing anything

**Comment by yaaHosein** [comment]
image

**Comment by yaaHosein** [comment]
image

**Comment by yaaHosein** [comment]
image

**Comment by yaaHosein** [comment]
image

**Comment by yaaHosein** [comment]
image

**Comment by SMH110** [comment]
this merged now, next you need to be extra careful when it come to creating branches so you don't end up wasting your time on fixing conflicts

**Comment by yaaHosein** [comment]
ok thanks !

**Pull request merged**